console/src/api/ApiHeader.tsx

```diff
@@ -76,16 +76,8 @@ function ApiHeader() {
                     return Promise.reject(error)
                 }
 
-                // allow non-exist model file in the eval result page
-                if (
-                    error.response?.status === 404 &&
-                    error.response.config.params?.partName?.includes('eval_panel_layout.json')
-                ) {
-                    return Promise.reject(error)
-                }
-
-                // use user/current as default token auth, it will be triggered multi times, so silent here
-                const withSilentRoute = error.response?.config.url.includes('/user/current')
+                // for example: user/current as default token auth, it will be triggered multi times, so silent here
+                const withSilentRoute = error.response.config.params.silent
```

Comment:
`params` will be sent to the server, use hash mark instead?

Reply:
Server will not handle this parameter, so I think it's will be ok ?